### DISCUSSION TITLE
SeQUeNCe Installation

### DISCUSSION THREAD

**Discussion by Emrissa** [sequence-toolbox/SeQUeNCe#154] [Q&A] [answered]
Hi,

I have a few questions:
1. Where can I refer to for the installation process? 
2. Can we use SeQUeNCe to simulate decoherence process?

Thank you.

**Comment by Alex-Kolar** [comment, marked as answer]
Hi Emrissa,

Glad to hear you're interested in using SeQUeNCe. To install the simulator, please refer to [these instructions](https://github.com/sequence-toolbox/SeQUeNCe#installing). The base version is fully written in python, so there should be minimal requirements other than python version 3.8+. SeQUeNCe can also be installed within a virtual environment or conda environment, if desired.

To simulate decoherence processes, the current production version provides minimal support for decoherence processes, only looking at a single floating-point fidelity. However, we have recently been working on decoherence processes more concretely in the Fock basis over on the [memory_simulation branch](https://github.com/sequence-toolbox/SeQUeNCe/tree/memory_simulation), specifically for excitation decay. The code for this may be found on line 677 of `src/kernel/quantum_manager.py`.

Let me know if you have further questions, I'm happy to help.

Thanks,
Alex
